packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/start-job-run.test.ts

```diff
@@ -0,0 +1,1061 @@
+          'emr-containers:CancelJobRun',
+        ],
+        Effect: 'Allow',
+        Resource: {
+          'Fn::Join': [
+            '',
+            [
+              'arn:',
+              {
+                Ref: 'AWS::Partition',
+              },
+              ':emr-containers:',
+              {
+                Ref: 'AWS::Region',
+              },
+              ':',
+              {
+                Ref: 'AWS::AccountId',
+              },
+              ':/virtualclusters/*',
+            ],
+          ],
+        },
+      }],
+    },
+  });
+});
+
+test('Permitted role actions for Start Job Run with REQUEST/RESPONSE integration pattern', () => {
+  // WHEN
+  const task = new EmrContainersStartJobRun(stack, 'Task', {
+    virtualClusterId: sfn.TaskInput.fromText(clusterId),
+    releaseLabel: ReleaseLabel.EMR_6_2_0,
+    jobDriver: {
+      sparkSubmitJobDriver: {
+        entryPoint: sfn.TaskInput.fromText('job-location'),
+      },
+    },
+    integrationPattern: sfn.IntegrationPattern.REQUEST_RESPONSE,
+  });
+
+  new sfn.StateMachine(stack, 'SM', {
+    definition: task,
+  });
+
+  // THEN
+  expect(stack).toHaveResourceLike('AWS::IAM::Policy', {
+    PolicyDocument: {
+      Statement: [{
+        Action: 'emr-containers:StartJobRun',
+        Condition: {
+          StringEquals: {
+            'emr-containers:ExecutionRoleArn': {
+              'Fn::GetAtt': [
+                'TaskJobExecutionRole5D5BBA5A',
+                'Arn',
+              ],
+            },
+          },
+        },
+        Resource: {
+          'Fn::Join': [
+            '',
+            [
+              'arn:',
+              {
+                Ref: 'AWS::Partition',
+              },
+              ':emr-containers:',
+              {
+                Ref: 'AWS::Region',
+              },
+              ':',
+              {
+                Ref: 'AWS::AccountId',
+              },
+              `:/virtualclusters/${clusterId}`,
+            ],
+          ],
+        },
+      }],
+    },
+  });
+
+});
+
+test('Permitted IAM policy actions for EMR Containers SDK call for Describe Virtual Cluster', () => {
+  // WHEN
+  const task = new EmrContainersStartJobRun(stack, 'EMR Containers Start Job Run', {
+    virtualClusterId: sfn.TaskInput.fromText(clusterId),
+    releaseLabel: ReleaseLabel.EMR_6_2_0,
+    jobDriver: {
+      sparkSubmitJobDriver: {
+        entryPoint: sfn.TaskInput.fromText('local:///usr/lib/spark/examples/src/main/python/pi.py'),
+        sparkSubmitParameters: '--conf spark.executor.instances=2',
+      },
+    },
+  });
+
+  new sfn.StateMachine(stack, 'SM', {
+    definition: task,
+  });
+
+  // THEN
+  expect(stack).toHaveResourceLike('AWS::IAM::Policy', {
+    PolicyDocument: {
+      Statement: [{
+        Action: 'emr-containers:DescribeVirtualCluster',
+        Resource: '*',
+      }],
+    },
+  });
+});
+
+test('Permitted IAM policy permissions for Custom Resource Lambda', () => {
```

Comment:
replace this test with one that confirms that a CustomResource is created that has the correct EKS namespace environment, CLI layer, and permissions